package.json

```diff
@@ -22,6 +22,12 @@
       "git add"
     ]
   },
+  "semistandard": {
+    "globals": [
+      "CRM",
+      "jQuery"
```

Comment:
why is jQuery needed? Aren't we following this IIFE-based format everywhere
```js
(function ($) {

}(CRM.$));
```
? If not, any reason you can see for it?

Reply:
In `js/noConflict` file we are using `jQuery`
```
// https://civicrm.org/licensing
(function($) {
  $.fn.button.noConflict();
})(jQuery);
```